src/docfx/build/page/Markup.cs

```diff
@@ -99,7 +95,18 @@ public static (MarkdownDocument ast, MarkupResult result) Parse(string content)
                     t_dependencyMap = dependencyMap;
                     t_bookmarkValidator = bookmarkValidator;
                     t_buildChild = buildChild;
-                    var html = Markdown.ToHtml(markdown, s_pipelineMapping[pipelineType]);
+                    if (t_pipelineMapping == null)
```

Comment:
just `t_xrefMap`, keep `s_pipelineMapping`

Reply:
we couldn't keep s_pipelineMapping any more, since the creation relies on input xrefmap

---
In reply to: [218298772](https://github.com/dotnet/docfx/pull/3428#discussion_r218298772) [](ancestors = 218298772)